lib/charliecloud.py

```diff
@@ -427,7 +429,8 @@ def FATAL(msg, hint=None, **kwargs):
 
 def INFO(msg, hint=None, **kwargs):
    "Note: Use print() for output; this function is for logging."
-   log(msg, hint, None, "33m", "", **kwargs)  # yellow
+   if (log_quiet == 0):
```

Comment:
I think things would be simpler if there was a single verbosity variable. Quiet could be negative values. Many programs call this “log level” ([e.g.](https://www.javatpoint.com/log4j-logging-levels)).

Also I don't recall whether there's an enum for it, but there probably should be, to avoid magic numbers. Then it would read something like:

```suggestion
   if (verbose >= LOG_INFO):
```